src/pages/tutorials/flutter-wallet.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: Create a Chain Agnostic Web3 wallet in Flutter
+title: Create a chain-agnostic web3 wallet in Flutter
 image: 'img/tutorials/tutorials-banners/flutter-wallet.png'
-description: Empower your Flutter app with a Chain Agnostic Web3 wallet using the Web3Auth PnP SDK.
```

Comment:
Should this be MetaMask Embedded Wallets PnP SDK?

Reply:
The tutorials haven't been updated with the new terminology, presumably because the dashboard is not updated. So we can keep as is until the dashboard changes.